person.rb

```diff
@@ -1,14 +1,25 @@
-class Person
+class Nameable
+  def correct_name
+    raise NotImplementedError, "#{self.class} must implement the correct_name method"
+  end
+end
```

Comment:
[OPTIONAL] Please create one file called `nameable.rb` and move lines no. 1 to 5 in this newly created file